### PR TITLE
Update Terraform setup from Mike's journey

### PR DESCRIPTION
**Pull request by sethvargo**
- Fixes GH-214

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -352,11 +356,40 @@ resource "google_cloud_run_service" "exposure" {
   depends_on = [null_resource.submit-build-and-publish, google_project_service.services["run.googleapis.com"], google_project_service.services["sqladmin.googleapis.com"]]
 }
 
+resource "google_service_account" "export" {
+  project      = data.google_project.project.project_id
+  account_id   = "en-export-sa"
```
Comment (mikehelmick) [inline terraform/main.tf:361]: this SA is missing secret accessor

File: terraform/main.tf [hunk 2]
```diff
@@ -352,11 +356,40 @@ resource "google_cloud_run_service" "exposure" {
...
+
+  secret_id = google_secret_manager_secret.db-pwd.id
+  role      = "roles/secretmanager.secretAccessor"
+  member    = "serviceAccount:${google_service_account.export.email}"
+}
+
+resource "google_storage_bucket_iam_member" "export-objectadmin" {
+  bucket = google_storage_bucket.export.name
+  role   = "roles/storage.objectAdmin" // overwrite is not included in objectCreator
+  member = "serviceAccount:${google_service_account.exposure.email}"
```
Comment (mikehelmick) [inline terraform/main.tf:382]: this should be : export.email

File: terraform/main.tf [hunk 3]
```diff
@@ -82,7 +72,7 @@ resource "google_sql_database_instance" "db-inst" {
   database_version = "POSTGRES_11"
   name             = "contact-tracing-${random_string.db-name.result}"
   settings {
-    tier              = "db-custom-1-3840" # "db-custom-32-122880"
+    tier              = var.cloudsql_tier
     availability_type = "REGIONAL"
     disk_size         = 500
```
Comment (crwilcox) [inline terraform/main.tf:77]: could you add `var.cloudsql_disk_size`?

Comment (sethvargo) [reply terraform/main.tf:77]: Done

File: terraform/main.tf (hunk 2 above)
Comment (sethvargo) [reply terraform/main.tf:382]: Fixed

**Pull request merged**